### DISCUSSION TITLE
New icon for unlinking

### DISCUSSION THREAD

**Discussion by PatriRoman** [Telefonica/mistica-design#1280] [Proposals]
### Goal

¡Hola!

Os traigo una necesidad y petición de mejora:
Estamos trabajando en una nueva funcionalidad en CMS que consiste en desasignar módulos de layouts (layout entendido como un paquete, un conjunto de módulos y audiencias juntas). 

Poder deslinkar un módulo de un layout, pero no borrarlo, no eliminar del CMS ese módulo, sólo de esa lista de módulos pertenecientes a un layout.

Esta acción irá dentro de un kebap de acciones junto con otras, que tienen un icono de apoyo para entender mejor la función.
Ahora mismo para esa acción de "desasignar/deslinkar/deavincular" he puesto el icono de link:
<img width="297" alt="image" src="https://github.com/Telefonica/mistica-design/assets/108683238/574077e0-3e92-49d0-b636-32543cab7acd">

Pero tendría mucho más sentido poner el icono de link roto, no? Algo como esto (es un boceto rápido para que entendáis a qué me refiero):
<img width="457" alt="image" src="https://github.com/Telefonica/mistica-design/assets/108683238/0929a790-9a3a-42b0-bf9e-d7945471476b">

¿Qué opináis? 

¡Gracias!

### Context

Nueva funcionalidad de CMS que consiste en desvincular módulos de layouts sin eliminarlos del repositorio de módulos de Explore dentro del CMS.

Dentro de cada layout, tenemos un apartado de "modules" en donde vemos los módulos asignados a ese layout. Podemos añadir más, editar los existentes, publicarlos (si no están publicados ya) y se añade esta nueva opción de "unassign" para la cual creemos que necesitamos ese nuevo icono.
<img width="749" alt="image" src="https://github.com/Telefonica/mistica-design/assets/108683238/b668133c-afff-45ab-9aad-6196c5a73e4d">

### Impact

El impacto será interno, puesto que aparecerá sólo en CMS.

### Discussion Questions

- Os parece buena solución este icono nuevo que proponemos?
- Consideráis que se entiende mejor el ya existente de "link"?
- Se os ocurre otro icono que pueda ilustrar mejor esta acción de "unassign"? 

### Do you have a visual proposal?

https://www.figma.com/file/0lecUlrNJGZBSqLvBCiCx2/Cards-Configuration?type=design&node-id=7816%3A99225&t=7Ij6pjr2HfTERpZ8-1

**Comment by yceballost** [comment]
Estaría guay tener un pequeño bench de capturas de cómo en nounproject u otros servicios (feather icons, iconmnstr, etc) suelen representar este concepto. Lejos de copiar, creo que es una buena forma de acercarnos a un estándar. 

No tanto para hacerlo igual, sino para saber qué tipo de recurso utilizan para un icono "contrario"

- por ejemplo, un lapiz que no es editable, alguna vez he visto que se hace un "corte", debería de ser algo parecido aquí? 
- hay algún icono en mistica icons que ya use un recurso parecido?

![image](https://github.com/Telefonica/mistica-design/assets/6722153/f9c89778-4ce9-4946-9e2a-6ab534d34b5a)

**Reply by PatriRoman** [reply]
Si, justo estuve viendo cuando empezamos con esta petición de unassign cómo lo representaban en nounproject. Basicamente lo hacen con ese corte como la bell, con separación literal de las dos partes de enlace o con esas líneas que representan la rotura.
Aquí os dejo varios ejemplos que vi de maneras diferentes de representarlo:
<img width="690" alt="image" src="https://github.com/Telefonica/mistica-design/assets/108683238/fe33da5f-472d-4902-8d4f-974ebddb0b39">

**Reply by yceballost** [reply]
Guay! pues si nos pasas los 3 pesos lo incluimos asap

**Comment by aweell** [comment]
@PatriRoman este tema en qué se quedó?

**Reply by PatriRoman** [reply]
@aweell  Pues no llegué a hacer los iconos. Puedo ponerme una tarea en el backlog para hacer el icono en los 3 pesos

**Reply by AnaMontes11** [reply]
Más que nada por ti si sigues necesitándolos ;)
